lib/smart_answer_flows/student-finance-forms.rb

```diff
@@ -73,8 +73,12 @@ def define
       end
 
       multiple_choice :what_year_full_time? do
-        option 'year-1617'
-        option 'year-1516'
+        question_value = self
+        precalculate :setup_options do
+          question_value.option "year-1718" if type_of_student == "uk-full-time"
+          question_value.option "year-1617"
+          question_value.option "year-1516" if type_of_student == "eu-full-time"
+        end
 
```

Comment:
@ikennaokpala The commit heading needs to be changed from `2105/16` to `2015/16' :)

I would have split this into two commits:

1) Add `2017-2018` as an option for `uk-full-time` students, and adds new outcome templates that match the option name and adds new test artefacts.
2) Remove `2015-2015` as an option for `uk-full-time` students and removes test artefacts.

That would make it easier to follow what's going on a bit.